frontend/package.json

```diff
@@ -58,7 +58,6 @@
     "ol": "^6.14.1",
     "ol-ext": "^3.2.22",
     "orbit-controls-es6": "^2.0.1",
-    "platform-detect": "^3.0.1",
```

Comment:
Remove platform-detect, because couldn't figure out in a reasonable amount of time how to resolve typescript errors associated with it when running jest tests.